src/webchat/store/messages/message-middleware.ts

```diff
@@ -93,6 +93,17 @@ export const createMessageMiddleware = (client: SocketClient): Middleware<{}, St
             const isMessageEmpty = !(message.text || message.data?._cognigy?._webchat);
             const isUnseen = !isWebchatActive && !isMessageEmpty;
 
+            if(store.getState().config.settings.useQuickReplyTitleAsPostback) {
```

Comment:
this should rather be handled in the messenger plugin rather than at a central place.

Reply:
Thx, fixed it